src/components/StaleBundleWarning.js

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from 'react-query';
+import ky from 'ky';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Button, MessageBanner } from '@folio/stripes-components';
+import { useStripes } from '../StripesContext';
+
+const StaleBundleWarning = () => {
+  const stripes = useStripes();
+  const config = stripes?.config?.staleBundleWarning;
+  const refetchInterval = (config?.interval ?? 15) * 60 * 1000;
+  const [previous, setPrevious] = useState();
+  const [stale, setStale] = useState(false);
+  const query = useQuery({
+    queryKey: ['StaleBundleWarning'],
+    queryFn: async () => {
+      if (typeof config?.path !== 'string') return null;
+      try {
+        if (typeof config?.header === 'string') {
+          const res = await ky(config.path, { method: 'head' });
+          return res.headers.get(config.header);
+        } else {
+          return await ky.get(config.path).text();
+        }
```

Comment:
It would be great if you provided an example config object in a header comment to this component, explaining what it does and how to use `stripes.config.js` to use it. You provided a nice [explanation and screenshot on #slack](https://folio-project.slack.com/archives/CAN13SWBF/p1646060923092559), but that information will be harder to find when this is less front-of-mind than it is right now.

Reply:
I was intending to put the docs at the top of the PR in the dev guide where we have the other config options but they're in a really confusing place there and discussed on Slack, we really need a proper document for Stripes deployment rather than development which is where this really belongs. And an example would be nice indeed.

So for now it can live inline on the file as you suggest.